lib/tasks/publishing_api.rake

```diff
@@ -32,6 +32,56 @@ namespace :publishing_api do
       ))
     end
   end
+
+  desc "Send publishable item links to Publishing API"
+  task publishing_api_patch_links: :environment do
+    $stdout.puts "# Sending all items' links to Publishing API"
+    $stdout.puts "# Ruby code is output for failed operations"
+    $stdout.puts
+
+    def patch_links(item)
+      content_id = item.content_id
+      links = PublishingApiPresenters.presenter_for(item).links
+      if links
+        Whitehall.publishing_api_v2_client.patch_links(content_id, {links: links})
+      else
+        $stdout.puts "# No links for item with content_id: #{content_id}"
+      end
+    rescue => err
+      message = "# Class: #{item.class}; id: #{item.id}; FAILURE: #{err.message}"[0..99]
+      $stderr.puts message
+      $stderr.puts("#" * message.length)
+      payload = { links: links } if links
+      $stderr.puts "Whitehall.publishing_api_v2_client.patch_links('#{content_id}', #{payload})" if links
+      $stderr.puts
+    end
+
+    [
+      Unpublishing,
+      AboutPage,
+      Classification,
+      MinisterialRole,
+      Organisation,
+      Person,
+      PolicyGroup,
+      TakePartPage,
+      WorldLocation,
+      StatisticsAnnouncement,
+    ].each do |klass|
+      count = klass.all.count
+      $stdout.puts "# Sending #{count} #{klass} items to Publishing API"
+      $stdout.puts
+      klass.all.each do |publishable_item|
```

Comment:
`klass.find_each` will do the retrieval in batches, so that we don't have to load all objects into memory before the loop.